api/lib/schema/types.ex

```diff
@@ -17,4 +17,9 @@ defmodule Api.Schema.Types do
     field(:name, :string)
     field(:sets, list_of(:set))
   end
+
+  input_object :set_input do
+    field(:exercise, non_null(:exercise))
```

Comment:
I think this should be a different object than `:exercise` - try going to `/api/graphiql`, it can't parse the schema and crashes. Maybe we need a separate exercise_input object ?